### PR TITLE
Add products reviews to schema.org

### PR DESCRIPTION
**Pull request by rodriciru**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add customer reviews to product json+ld
| Type?             | improvement
| BC breaks?        |  no
| Deprecations?     |  no

This change depends on this pull request, as we have the data from somewhere:
https://github.com/PrestaShop/productcomments/pull/140

But for now i add the code to print the reviews

### CODE REVIEW

File: templates/_partials/microdata/product-jsonld.tpl [hunk 1]
```diff
@@ -17,6 +17,26 @@
       "@type": "Brand",
       "name": "{if $product_manufacturer->name}{$product_manufacturer->name|escape:'html':'UTF-8'}{else}{$shop.name}{/if}"
     },{/if}
+    {if !empty($product.productComments.comments)}
```
Comment (matks) [inline templates/_partials/microdata/product-jsonld.tpl:20]: As I mentioned in the github issue I think the idea is good but also I think we should try to expose a variable that is NOT related to productcomment module. This allows people to remove the productcomment module if they want to and to be able to inject this variable.

Something like `product.reviewData` ?

Comment (NeOMakinG) [reply templates/_partials/microdata/product-jsonld.tpl:20]: I guess it will even throw a 500 in case the module has been deleted nop?

**Review by matks — CHANGES_REQUESTED** [review]

File: templates/_partials/microdata/product-jsonld.tpl (hunk 1 above)
Comment (Hlavtox) [reply templates/_partials/microdata/product-jsonld.tpl:20]: We should say how the structure should look, and review modules can include it in hookProductLazyArray.

Comment (rodriciru) [reply templates/_partials/microdata/product-jsonld.tpl:20]: > @rodriciru you need to check that the variable is defined too

If I uninstall, or disable the module, nothing bad is happening. Simply it ignores the comments part and the review, but it won't break

**Comment by rodriciru** [comment]
Anything i can do?. I dont know if i close the requested changes or what. I new to this github part

**Review by NeOMakinG — DISMISSED** [review]
I guess it's acceptable for now, but I would like the @Hlavtox advice as he commented something

**Comment by Hlavtox** [comment]
@NeOMakinG @rodriciru @matks Everything is good with the JSON work, I approve, but let's use the moment to think about unifying the structure.

What if we just require all review modules to provide data in standardized format? So if you swap a review module, all will work.
`$product.rating.reviews`
`$product.rating.total`
`$product.rating.average`

**Comment by rodriciru** [comment]
> @NeOMakinG @rodriciru @matks Everything is good with the JSON work, I approve, but let's use the moment to think about unifying the structure.
> 
> What if we just require all review modules to provide data in standardized format? So if you swap a review module, all will work. `$product.rating.reviews` `$product.rating.total` `$product.rating.average`

I love standardization.

**Comment by matks** [comment]
I agree unifying would be good 👍 and `product.rating` looks good enough for me to be used right now

**Comment by NeOMakinG** [comment]
@Hlavtox so we need to tweak the module itself first?

**Comment by aniszr** [comment]
Hello,
Could you please resolve this branch conflicts and apply the requested changes.

Thanks!

**Comment by matks** [comment]
Hi @ga-devfront can you tell us why you closed this PR? Is it not relevant anymore

Merging this PR was a blocker for PR https://github.com/PrestaShop/productcomments/pull/140

**Comment by ga-devfront** [comment]
> Hi @ga-devfront can you tell us why you closed this PR? Is it not relevant anymore
> 
> Merging this PR was a blocker for PR [PrestaShop/productcomments#140](https://github.com/PrestaShop/productcomments/pull/140)

My apologies, it was a handling error on my part when I set up the work for PrestaShopCorp...
